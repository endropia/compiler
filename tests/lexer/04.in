A
a
a_
a2
2a
_a
____
a_23