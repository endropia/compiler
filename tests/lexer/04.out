1	1	Identifier	A	A
2	1	Identifier	a	a
3	1	Identifier	a_	a_
4	1	Identifier	a2	a2
5	1	Integer	2	2
5	2	Identifier	a	a
6	1	Identifier	_a	_a
7	1	Identifier	____	____
8	1	Identifier	a_23	a_23
8	5	eof
